client/state/signup/verticals/reducer.js

```diff
@@ -4,14 +4,22 @@
  * Internal dependencies
  */
 import { createReducer } from 'state/utils';
-import { SIGNUP_VERTICALS_SET } from 'state/action-types';
+import { SIGNUP_VERTICALS_SET, SIGNUP_VERTICALS_REQUEST } from 'state/action-types';
 
 const verticals = createReducer(
 	{},
 	{
+		[ SIGNUP_VERTICALS_REQUEST ]: state => {
+			return {
+				...state,
+				isFetching: true,
```

Comment:
I usually advise against any `isFetching`/`isRequesting` when possible. If you're sure we need it and there's no other way - that's fine.

Reply:
Thanks for raising this. Normally I'd have not added this to the state. There was some headscratching around the problem.

I should be able to wrangle it, but I'll need to add a custom prop method, something like:

```
isVerticalSearchPending: ( searchValue, verticals = [] ) => searchValue && 0 === verticals.length,
```

Or something... :)